set_fm_data.py

```diff
@@ -81,7 +81,20 @@ def initialize(self):
         # ToDo: AJO 2022-12-30: This code is copied in several modules: combine!
         self.CAR_MIN_SOC_IN_PERCENT = int(float(self.args["car_min_soc_in_percent"]))
         # Make sure this value is between 10 en 30
-        self.CAR_MIN_SOC_IN_PERCENT = max(min(30, self.CAR_MIN_SOC_IN_PERCENT), 10)
+        notification_message = ""
+        if self.CAR_MIN_SOC_IN_PERCENT < 10:
+            notification_message = f"Setting for minimum SoC (car_min_soc_in_percent) {self.CAR_MIN_SOC_IN_PERCENT} " \
+                                   f"in secrets.yaml too low. Using minimum value 10."
+            self.CAR_MIN_SOC_IN_PERCENT = 10
+        elif self.CAR_MIN_SOC_IN_PERCENT > 30:
+            notification_message = f"Setting for minimum SoC (car_min_soc_in_percent) {self.CAR_MIN_SOC_IN_PERCENT}" \
+                                   f" in secrets.yaml too high. Using maximum value 30."
+            self.CAR_MIN_SOC_IN_PERCENT = 30
+
+        if notification_message != "":
```

Comment:
Slightly easier to read and equivalent: `if notification_message:`

Reply:
Agree, will change